src/lib/containers/download_list/DownloadDetails.tsx

```diff
@@ -31,13 +31,14 @@ type State = {
  * @returns
  */
 export default function DownloadDetails(props: DownloadDetailsProps) {
+  const { numFiles, numBytes } = props
+  const { accessToken } = useSynapseContext()
+
   const [state, setState] = useState<State>({
-    isLoading: true,
+    isLoading: !!numBytes, // loading=true (to figure out the estimated download time) iff we were given a byte count
```

Comment:
Just dropping old code from the comment, but keeping the useful part
```suggestion
    isLoading: !!numBytes, // figure out the estimated download time iff we were given a byte count
```